app/js/components/orderEntry/addForm/DraftDataTable.jsx

```diff
@@ -152,9 +152,14 @@ export class DraftDataTable extends React.Component {
       </p>
     );
 
+    let formattedDetails = details.props.children.join("");
+    if (formattedDetails.length > 155) {
+      formattedDetails = `${formattedDetails.substring(0, 155)}...`;
+    }
+
     const discontinueForm = (
       <div>
-        <p> Discontinue {drugName} </p>
+        <p className="discontinue-drug"><em>{formattedDetails}</em></p>
```

Comment:
You need to add the hard coded "Discontinue" part

Reply:
I have a feeling that this is not necessary since the status column beside this column already denotes that. This will also maintain consistency with the rest of the items on the DraftTable